manifest/armv7l/b/bubblewrap.filelist

```diff
@@ -1,5 +1,4 @@
 /usr/local/bin/bwrap
 /usr/local/bin/bwrap.elf
 /usr/local/share/bash-completion/completions/bwrap
-/usr/local/share/man/man1/bwrap.1.zst
```

Comment:
Removing manpages always makes me sad.

Reply:
Agreed. But if there's a build issue with XML file weirdness, which there was here...

Reply:
Kinda makes me wonder if it's worth changing the build method.

Reply:
> Kinda makes me wonder if it's worth changing the build method.

For bubblewrap? I doubt it. There is extensive documentation online. Most people are going to Google first...

Reply:
Well, the old method did build the manpage or does it not with this new release?

Reply:
Man is set to auto automatically with this release. So if there's an issue in build setup, it won't bother building it.

Reply:
We could manually turn it on...

Reply:
Ah! I broke `docbook_xsl` in August.... Fixing it...

Reply:
Fixed.